install_manual.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+echo "Installing JetBrains Mono Fonts"
+echo ""
+# Get the download URL of the latest release from Github
+FONT_URL=$(curl -s https://api.github.com/repos/JetBrains/JetBrainsMono/releases/latest | grep browser_download_url | cut -d '"' -f 4)
```

Comment:
This will not work if a new asset will be added. 🤔

Reply:
The script will be modified then, I believe 👀  As of now, there is only one asset 🤷🏻‍♂️

Reply:
Fair enough :) Unfortunately `jq` is not bundled in modern distributives.

Actually I wonder if setting up github action that will embed asset link might be an option here.

Reply:
Where did I use `jq`🤔?

It uses the most basic and bundled tools on any linux distribution. I.e `curl`. `grep`, `cut` and `unzip`.

This is a a simple script.

I am not sure why would you want Github Actions setup here 🤔 Why overcomplicate things when they don't need to be?

Reply:
> Where did I use jqthinking?

Nowhere. My point was that `jq` is very handy for json processing in shell but it's not widespread and (unfortunately) can't be used in this case.

> I am not sure why would you want Github Actions setup here thinking Why overcomplicate things when they don't need to be?

Just making suggestions as current parsing method seems a bit hacky for me. :)
But I'm not a decision maker in this repo.